### DISCUSSION TITLE
Looking for a way to set vd's color scheme to black-on-white

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#1354] [General]
I'm looking for a way to set vd's color scheme to black-on-white (i.e. a 'light' scheme). I saw the `color_default` option, but I guess all the different `color_..` options have to be adjusted together. 

Does anyone use a light black-on-white color scheme?

**Comment by saulpw** [comment]
Here, I made a simple one tonight:

    vd.options.color_default      = 'black on white'  # the default fg and bg colors
    vd.options.color_key_col      = '20 blue'   # color of key columns
    vd.options.color_edit_cell    = '234 black'     # cell color to use when editing cell
    vd.options.color_selected_row = '164 magenta'  # color of selected rows
    vd.options.color_note_row     = '164 magenta'  # color of row note on left edge
    vd.options.color_note_type    = '88 red'  # color of cell note for non-str types in anytype columns
    vd.options.plot_colors = '20 red magenta black 28 88 94 99 106'

It's available in [vdplus](https://github.com/visidata/vdplus/blob/develop/lightscheme.py) with `vd.use_light_colors()`.

**Reply by halloleo** [reply]
This is cool, @saulpw ! Thanks heaps! (And sorry for the late reply. - I was on my first past(?)-pandemic travel trip.)

**Reply by victorlin** [reply]
I'm not sure if it's the same light theme, but this can be set for `vd` by adding `options.theme = 'light'` to `.visidatarc`.

EDIT: Found the commit 873eb6287279c4e3edfe1c4f82321dafa5aa9acd - looks like it's based on this initial implementation.

**Comment by 3d7l** [comment]
I would be very interested to use visidata with a light theme.
Unfortunately, I faced this error when cloning the repo:
$ git clone git@github.com:saulpw/vdplus ~/.visidata/vdplus
Cloning into '/home/bobby/.visidata/vdplus'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Would it be possible to use the light theme without vdplus ?
for example, by editing and setting colors directly to this file:
 /usr/lib/python3/dist-packages/visidata/vdtui.py
?

My config:
Ubuntu Release: 20.04
saul.pw/VisiData v1.5.2

**Reply by anjakefala** [reply]
Hi @crcmt!

You are running into challenges because that is a very old version of VisiData! The current version is v2.11. vdtui.py does not exist anymore. 

If you upgrade to the current `develop` version, it actually has a command `theme-input` for more easily setting the light-mode theming. 

To use a command from its longname use the `Spacebar`. So the full keystrokes are:

`Spacebar theme-input`
`light`.

**Comment by 3d7l** [comment]
Forgot to upgrade to the last version indeed !

I installed the dev version with:
pip3 install git+https://github.com/saulpw/visidata.git@develop

and added this line in ~/.visidatarc
<--
options.theme = "light"
-->                                                   

The light theme is now working.

Thanks a lot !

**Reply by anjakefala** [reply]
Let us know if something feels Not Good about it!
